larq_compute_engine/mlir/tf_tfl_passes.cc

```diff
@@ -20,7 +20,8 @@ namespace tensorflow {
 void AddQuantizationPasses(const mlir::TFL::QuantizationSpecs& quant_specs,
                            mlir::OpPassManager* pass_manager) {
   pass_manager->addPass(mlir::TFL::CreatePrepareQuantizePass(quant_specs));
-  pass_manager->addPass(mlir::TFL::CreateHybridQuantizePass());
+  pass_manager->addPass(mlir::TFL::CreateLCEQuantizePass());
+  pass_manager->addPass(mlir::TFL::CreateQuantizePass());
```

Comment:
Just out of curiosity, does the order of `LCEQuantizePass` and `QuantizePass` matter? Since our op is in a different dialect, the `QuantizePass` should leave it alone right?

Reply:
> Just out of curiosity, does the order of LCEQuantizePass and QuantizePass matter? 

If I do it the other way around I ran into some problems where the multiplier and bias were casted to int8 (since we only remove the quantize dequantize pairs in the lce quantize pass), which now fails the op verification in our dialect.

> Since our op is in a different dialect, the QuantizePass should leave it alone right?

The quantize pass will also run on our op as well, the different dialect is not a problem.